src/icons/folderIcons.ts

```diff
@@ -184,6 +184,7 @@ export const folderIcons: FolderTheme[] = [
             { name: 'folder-mobile', folderNames: ['mobile', 'mobiles', 'portable'] },
             { name: 'folder-stencil', folderNames: ['.stencil'] },
             { name: 'folder-firebase', folderNames: ['.firebase'] },
+            { name: 'folder-emails', folderNames: ['email', 'emails'] },
```

Comment:
Yes, @devcass is right, there's already a mail folder icon. It would be great if you could update your pull request so that the icons will be removed and the additional folder names "email" and "emails" are added to the existing mail folder associations.